### PR TITLE
Search results table now shows the Ref for each item

### PR DESCRIPTION
**Pull request by artlepool**
Related to pull request 337 on taiga-back, although they are not dependent on each other.

This pull request adds the Ref number as a column on the search results for user stories, issues & tasks. 
Also added REF in to the English locale file - its beyond my skill set to translate to other languages I'm afraid :-)

### CODE REVIEW

File: app/partials/includes/modules/search-result-table.jade [hunk 1]
```diff
@@ -4,11 +4,13 @@ script(type="text/ng-template", id="search-issues")
     div.search-result-table-container(ng-class="{'hidden': !issues.length}", tg-bind-scope)
         div.search-result-table-header
             div.row.title
+                div.ref(translate="COMMON.FIELDS.REF")
                 div.user-stories(translate="SEARCH.FILTER_ISSUES")
                 div.status(translate="COMMON.FIELDS.STATUS")
                 div.assigned-to(translate="COMMON.FIELDS.ASSIGNED_TO")
         div.search-result-table-body
             div.row.table-main(ng-repeat="issue in issues track by issue.id")
+                div.ref(tg-bo-bind="issue.ref")
```
Comment (bameda) [inline app/partials/includes/modules/search-result-table.jade]: Use ```tg-bo-ref="issue.ref"``` instead

File: app/partials/includes/modules/search-result-table.jade [hunk 2]
```diff
@@ -26,11 +28,13 @@ script(type="text/ng-template", id="search-userstories")
     div.search-result-table-container(ng-class="{'hidden': !userstories.length}", tg-bind-scope)
         div.search-result-table-header
             div.row.title
+                div.ref(translate="COMMON.FIELDS.REF")
                 div.user-stories(translate="SEARCH.FILTER_USER_STORIES")
                 div.status(translate="COMMON.FIELDS.STATUS")
                 div.points(translate="COMMON.FIELDS.POINTS")
         div.search-result-table-body
             div.row.table-main(ng-repeat="us in userstories track by us.id")
+                div.ref(tg-bo-bind="us.ref")
```
Comment (bameda) [inline app/partials/includes/modules/search-result-table.jade]: Use ```tg-bo-ref="us.ref"``` instead

File: app/partials/includes/modules/search-result-table.jade [hunk 3]
```diff
@@ -47,11 +51,13 @@ script(type="text/ng-template", id="search-tasks")
     div.search-result-table-container(ng-class="{'hidden': !tasks.length}", tg-bind-scope)
         div.search-result-table-header
             div.row.title
+                div.ref(translate="COMMON.FIELDS.REF")
                 div.user-stories(translate="SEARCH.FILTER_TASKS")
                 div.status(translate="COMMON.FIELDS.STATUS")
                 div.assigned-to(translate="COMMON.FIELDS.ASSIGNED_TO")
         div.search-result-table-body
             div.row.table-main(ng-repeat="task in tasks track by task.id")
+                div.ref(tg-bo-bind="task.ref")
```
Comment (bameda) [inline app/partials/includes/modules/search-result-table.jade]: Use ```tg-bo-ref="task.ref"``` instead

**Comment by bameda** [comment]
Good work, I like it :dancers:

**Comment by superalex** [comment]
Thank you very much @artlepool!

Could you please make the suggested changes by @bameda before we can merge the PR?

Regards!

**Comment by artlepool** [comment]
Done!!

**Comment by bameda** [comment]
Merged https://github.com/taigaio/taiga-front/commit/3b26cf12b2eb0354c89c581450a384fd4e9644e9

Thank you

**Pull request closed without merging**